packages/KTable/KTable.vue

```diff
@@ -108,6 +108,11 @@ table.k-table {
           text-decoration: underline;
         }
       }
+      button,
+      .k-button {
+        padding-top: 0;
+        padding-bottom: 0;
```

Comment:
re: our discussion offline, we should hook into a css var for this one. e.g. a new KButton var: `--KButtonPaddingY`.

Reply:
Added to button and replaced here. Works like a charm 🎉